src/cephadm/tox.ini

```diff
@@ -41,7 +41,9 @@ commands=pytest {posargs}
 
 [testenv:mypy]
 basepython = python3
-deps = mypy==0.901
+deps = 
+    mypy
+    -c{toxinidir}/../mypy-constrains.txt
```

Comment:
Is this tox being called with `-c ...`?

Reply:
that's a pip "constrains.txt" file that looks like a regular requirements.txt file, with the exception of that it doesn't force pip to install requirements, but only contrains the version number

Reply:
Ah, so it's `pip -c`! Thank you very much, sir! That makes sense. I've looked into the docs of mypy and tox to find out what `-c` does, but it didn't make much sense.